Orange/widgets/unsupervised/owtsne.py

```diff
@@ -169,12 +171,16 @@ def error(err):
                 error(self.Error.not_enough_rows)
             elif not self.data.domain.attributes:
                 error(self.Error.no_attributes)
-            elif not self.data.is_sparse() and \
-                    not np.sum(np.nanstd(self.data.X, axis=0)):
-                error(self.Error.constant_data)
             elif not self.data.is_sparse() and \
                     np.all(~np.isfinite(self.data.X)):
                 error(self.Error.no_valid_data)
+            elif not self.data.is_sparse():
```

Comment:
If we now have an `elif not sparse` then the above elif can be moved inside this one...